src/cmd/tools/archiver.go

```diff
@@ -32,12 +31,13 @@ var archiverCompressCmd = &cobra.Command{
 	Aliases: []string{"c"},
 	Short:   lang.CmdToolsArchiverCompressShort,
 	Args:    cobra.MinimumNArgs(2),
-	Run: func(_ *cobra.Command, args []string) {
+	RunE: func(_ *cobra.Command, args []string) error {
 		sourceFiles, destinationArchive := args[:len(args)-1], args[len(args)-1]
 		err := archiver.Archive(sourceFiles, destinationArchive)
 		if err != nil {
-			message.Fatalf(err, lang.CmdToolsArchiverCompressErr, err.Error())
+			return fmt.Errorf("unable to perform compression: %w", err)
```

Comment:
Well the question is really why we would? I would understand it if we were to export the errors so that users could check the error type. Otherwise it just adds complexity we do not need.

Reply:
consistency 🤷‍♀️